### PR TITLE
[service-configuration] Add types for ConfigError

### PR DESCRIPTION
**Pull request by harryadel**
Add types for configError 
https://github.com/meteor/meteor/blob/72f27da680cfdd3ff6fb20f580a7f468cb5f21ce/packages/service-configuration/service_configuration_common.js#L21

I'm not well versed in TS so a little help is appreciated.

### CODE REVIEW

**Comment by harryadel** [comment]
cc @piotrpospiech @radekmie @ebroder

File: packages/service-configuration/service-configuration.d.ts [hunk 1]
```diff
@@ -5,6 +5,12 @@ export interface Configuration {
   secret: string;
 }
 
+export interface ConfigError extends Error {
+  constructor(serviceName?: string);
+  message: string;
+}
+
```
Comment (ebroder) [inline packages/service-configuration/service-configuration.d.ts:12]: Hmm. This should probably be declared as a class, not as an interface

Comment (harryadel) [reply packages/service-configuration/service-configuration.d.ts:12]: Thanks, will apply that change. But a side question, whenever I make changes in TS definition file vscode isn't picking up on them, any idea why?

**Review by StorytellerCZ — APPROVED** [review]

File: packages/service-configuration/service-configuration.d.ts [hunk 2]
```diff
@@ -5,6 +5,12 @@ export interface Configuration {
...
 }
 
+class ConfigError extends Error {
+  constructor(serviceName?: string);
+  message: string;
+}
+
 export declare var ServiceConfiguration: {
   configurations: Mongo.Collection<Configuration>;
+  ConfigError
```
Comment (radekmie) [inline packages/service-configuration/service-configuration.d.ts:15]: Sorry for being late, but this is incorrect: with this syntax, `ConfigError` is `any`. It should say `ConfigError: typeof ConfigError` (without the `typeof` it'd be an instance and not the class itself).

**Pull request merged**